README.md

```diff
@@ -31,13 +31,13 @@ go build -o ./build/cognito
 Without client secret:
 
 ```go
-AWS_PROFILE=XXX COGNITO_APP_CLIENT_ID=XXX COGNITO_USER_POOL_ID=XXX PORT=8080 ./build/cognito
+AWS_REGION=XXX COGNITO_APP_CLIENT_ID=XXX COGNITO_USER_POOL_ID=XXX PORT=8080 ./build/cognito
 ```
 
 With client secret:
 
 ```go
-AWS_PROFILE=XXX COGNITO_APP_CLIENT_ID=XXX COGNITO_APP_CLIENT_SECRET=XXX  COGNITO_USER_POOL_ID=XXX PORT=8080 ./build/cognito
+AWS_REGION=XXX COGNITO_APP_CLIENT_ID=XXX COGNITO_APP_CLIENT_SECRET=XXX  COGNITO_USER_POOL_ID=XXX PORT=8080 ./build/cognito
```

Comment:
I think we still need `AWS_PROFILE` along with `AWS_REGION` change. Sometimes there are more profiles than one `default`, so that's why it was here. Could you please explain why did you remove it?

Reply:
The purpose of this repo is to provide the simplest possible example of how to use Cognito in Go, therefore `AWS_PROFILE` is needed. I think that docker is a bit of an advance technique and might confuse other devs who just want to check out how this repo works. Because of this, I propose to have both `AWS_PROFILE` and `AWS_REGION` in the README, like this:
```
AWS_PROFILE=XXX AWS_REGION=XXX COGNITO_APP_CLIENT_ID=XXX COGNITO_APP_CLIENT_SECRET=XXX  COGNITO_USER_POOL_ID=XXX PORT=8080 ./build/cognito
```